packages/admin-panel/src/routes/users/permissions.js

```diff
@@ -24,7 +24,14 @@ const EntityField = {
 };
 export const PERMISSIONS_ENDPOINT = 'userEntityPermissions';
 export const PERMISSIONS_COLUMNS = [
-  EntityField,
+  {
+    Header: 'Entity',
```

Comment:
```suggestion
    Header: 'Country',
```
?

Reply:
We use Entity on the new list field that is used on the create form so this needs to be Entity to be consistency.